### PR TITLE
boards: added support for the Calliope mini

### PR DESCRIPTION
**Pull request by haukepetersen**

### CODE REVIEW

File: boards/calliope-mini/Makefile.include [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# define the used CPU
+export CPU = nrf51
+export CPU_MODEL = nrf51x22xxab
+
+# define the default port depending on the host OS
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
```
Comment (kYc0o) [inline boards/calliope-mini/Makefile.include]: please change to `tty.usbmodem*`

Comment (haukepetersen) [inline boards/calliope-mini/Makefile.include]: done and squashed.

**Comment by haukepetersen** [comment]
forgot to blacklist the board for some tests and examples -> done.

**Comment by haukepetersen** [comment]
@kaspar030: any idea why Murdock is not being triggered again after pushing my latest commits?

**Comment by haukepetersen** [comment]
never mind, now it started...

**Comment by kYc0o** [comment]
And go!

**Pull request merged**